faq/tidb-lightning.md

```diff
@@ -120,3 +120,7 @@ With the default settings of 3 replicas, the space requirement of the target TiK
 
 - The space occupied by indices
 - Space amplification in RocksDB
+
+## Can TiKV-Importer be restarted while TiDB-Lightning is running?
+
+No, the Importer will store data of Engine in memeory. The Lightning must be restarted once the Importer is restarted.
```

Comment:
```suggestion
No, Importer stores data of engines in memory. Lightning must be restarted once Importer is restarted.
```

No need to add "the" before proper nouns.

Reply:
OK，addressed.
Thanks.